discovery-provider/src/models.py

```diff
@@ -153,6 +157,40 @@ class Track(Base):
     # Primary key has to be combo of all 3 is_current/creator_id/blockhash
     PrimaryKeyConstraint(is_current, track_id, blockhash)
 
+    # Will pass in all fields into @validates; hardcoded for now
+    @validates('title', 'length')
```

Comment:
where will these get passed in from? sorry unclear

Reply:
going out on a limb here, but this decorator will be applied to fields from the json schema that's yet to be defined? @vicky-g is that right?

Reply:
yes! i was thinking of creating the json schema, grabbing all the keys, and passing them into the `@validates` annotation